### DISCUSSION TITLE
迈特云是Vben的商业版吗？

### DISCUSSION THREAD

**Discussion by AnsGoo** [vbenjs/vue-vben-admin#1243] [Q&A] [answered]
看到一个项目，跟Vben很像但是是商业版——迈特云，是不是这个项目的商业版
github衔接： https://github.com/matevip/artemis， 
gitee衔接：https://gitee.com/matevip/artemis

**Comment by mynetfan** [comment, marked as answer]
vben目前没有商业版，所有收费项目都是第三方运作，与vben无关。



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2021年9月27日(周一) 上午10:49
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [anncwb/vue-vben-admin] 迈特云是Vben的商业版吗？ (Discussion #1243)




 
看到一个项目，跟Vben很像但是是商业版——迈特云，是不是这个项目的商业版
 github衔接： https://github.com/matevip/artemis，
 gitee衔接：https://gitee.com/matevip/artemis
